### PR TITLE
Bump v.0.2.7

### PR DESCRIPTION
**Pull request by nicolasgrieco**
- distro file included in the cookbook instead of fetching it from github

### CODE REVIEW

**Review by olive42 — DISMISSED** [review]
You should also remove the unused default['librenms']['snmp']['distro'] attribute.

What about the distro detection line in snmpd.conf.erb where the binary is referenced?

**Comment by nicolasgrieco** [comment]
As it is required after all, I've integrated it in the cookbook files.

File: files/default/distro [hunk 1]
```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env bash
```
Comment (jmauro) [inline files/default/distro:1]: Why not creating the file directly with chef since you have access the distribution definition?

**Review by jmauro — DISMISSED** [review]
3 commit and 2 commits have the same title.

File: files/default/distro (hunk 1 above)
Comment (nicolasgrieco) [reply files/default/distro:1]: Because it is unnecessary, as this one is working.

Comment (jmauro) [reply files/default/distro:1]: I don't understand we only support CentOS7 so we only have to handle CentOS 7.

**Comment by jmauro** [comment]
Too many commit, I don't understand why.

File: files/default/distro (hunk 1 above)
Comment (nicolasgrieco) [reply files/default/distro:1]: it works on debian/ubuntu also

**Pull request closed without merging**